docs/content/usage/issue-pull-request-templates.en-us.md

```diff
@@ -187,18 +194,27 @@ body:
       options:
         - label: I agree to follow this project's Code of Conduct
           required: true
+        - label: I also have read the CONTRIBUTION.MD
+          required: true
+          skip_submit: true
+        - label: This is a TODO only visible after issue creation
+          hide_on_form: true
 ```
 
 ### Markdown
 
-You can use a `markdown` element to display Markdown in your form that provides extra context to the user, but is not submitted.
+You can use a `markdown` element to display Markdown in your form that provides extra context to the user, but is not submitted by default.
 
 Attributes:
 
 | Key   | Description                                                  | Required | Type   | Default | Valid values |
 |-------|--------------------------------------------------------------|----------|--------|---------|--------------|
 | value | The text that is rendered. Markdown formatting is supported. | Required | String | -       | -            |
 
+skip_submit: Default is **true**
```

Comment:
I think this is a break change?

Reply:
github did start whith this that way :upside_down_face: